src/pages/LoginPage.js

```diff
@@ -1,9 +1,44 @@
 import React from 'react'
+import {ThemeButton} from "../components/ThemeButton";
+import { useForm } from "react-hook-form";
+import {TextField, Container, Grid, Box, Link} from "@material-ui/core";
+import { makeStyles } from '@material-ui/core/styles';
+import { Link as RouterLink } from 'react-router-dom';
+import {login} from "../mock/auth";
 
 export const LoginPage = () => {
+    const { register, handleSubmit, errors } = useForm();
+    const onSubmit = data => {
+        console.log(data);
+        login();
+    }
+    const classes = useStyles();
+
   return (
-    <div>
-      LoginPage
-    </div>
+      <Container className={classes.container} maxWidth="xs">
+          <form onSubmit={handleSubmit(onSubmit)}>
+              <Grid container
+                    direction="column"
+                    justify="space-between"
+                    alignItems="center"
+              >
+                  <Box><TextField label="Username" name="usernameLoginField" inputRef={register( {required: true})} /></Box>
+                  <Box marginTop={2}><TextField label="Password" name="passwordLoginField" inputRef={register({ required: true })} type="password" /></Box>
+                  <Box marginTop={5}><ThemeButton text={"log in"}/></Box>
+                  <Box marginTop={2}><Link component={RouterLink} to={"/register"}>Create an account.</Link></Box>
```

Comment:
Et alternativ her er å sette `spacing={2}` i `<Grid container ../>` og bruke `<Grid item>` på barna i stedet for `<Box >`. Da vi de automatisk få en fast avstand. Hvis en trenger mer på enkelte kan man legge det inn som `style={{paddingTop: 10}}` eller i with styles.

Begge deler fungerer uansett 🙂

Reply:
Aha, da gjør jeg det istedenfor :D